### PR TITLE
Add more user apis

### PR DESCRIPTION
**Pull request by JasonXian**
- Add User#CreateUser
- Add User#DeleteUser

### CODE REVIEW

**Comment by JasonXian** [comment]
@sedenardi please review when you have time!

File: src/users.ts [hunk 1]
```diff
@@ -76,10 +116,26 @@ export default function(zoomApiOpts: ZoomOptions) {
       params: params
     });
   };
+  const CreateUser = function(body?: CreateUserBody) {
```
Comment (sedenardi) [inline src/users.ts]: I believe `body` should be a required parameter according to the [docs](https://marketplace.zoom.us/docs/api-reference/zoom-api/users/usercreate).

**Review by sedenardi — CHANGES_REQUESTED** [review]

File: src/users.ts (hunk 1 above)
Comment (JasonXian) [reply src/users.ts]: Great catch, thanks!!

**Review by sedenardi — APPROVED** [review]
Looks good. I'll merge and comment when it's live on `npm`.

**Comment by sedenardi** [comment]
@JasonXian Version `1.3.0` is live on `npm`. Thanks again for your contributions!

**Pull request merged**